nixos/modules/config/networking.nix

```diff
@@ -187,11 +199,11 @@ in
         "rpc".source = pkgs.glibc.out + "/etc/rpc";
 
         # /etc/hosts: Hostname-to-IP mappings.
-        "hosts".text =
+        "hosts".text = let foo = concatStringsSep " " cfg.extraLocalHosts; in
```

Comment:
By naming this "variable" `foo` I meant that its name does not hold any significance in semantic space of whole program. It is just a technical variable for text that occurs in several (two in this case) places, an its purpose is minor increase in bug-pronness in a way that one will not change this text in one place and forget to do same in the other.

Its scope is limited to several lines, and its use is kinda obvious, so it should not be a problem (except greping source).

It is just a coding style thing, and I do not hold it dear. So if you propose how should be done, it should not be a problem at all.